src/Propel/Generator/Builder/Om/TableMapBuilder.php

```diff
@@ -444,6 +445,64 @@ protected function addFieldsAttributes()
         ]);
     }
 
+    /**
+     * @param string $script
+     *
+     * @return void
+     */
+    protected function addNormalizedColumnNameMap(&$script): void
+    {
+        $script .= '
+    /**
+     * Holds a list of column names to be normalized
```

Comment:
to be normalized?
How about
```suggestion
     * Holds a list of column names and their normalized version.
```
maybe?